databroker/src/broker.rs

```diff
@@ -1627,7 +1628,7 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
             return Err(SubscriptionError::InvalidInput);
         }
 
-        let (sender, receiver) = mpsc::channel(10);
+        let (sender, receiver) = broadcast::channel(10);
```

Comment:
Since we're moving to a model of dropping notifications when a subscriber lags, I think the buffer size should be configurable by the subscriber (i.e. the caller of `pub async fn subscribe(...)`)

In general, I think it's most likely that subscribers are only interested in receiving the latest value, and don't care whether an intermediate value was dropped (i.e. `broadcast::channel(1)`). But since there _can_ be other use cases, it should probably be configurable.

For the previous model, i.e. nothing was to be dropped, the buffer acted as a way to not immediately put back pressure on the provider.